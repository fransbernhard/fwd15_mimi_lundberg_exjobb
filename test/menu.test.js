import React, { Component } from 'react';
import { Router, Route, Link, browserHistory } from 'react-router';
import Scrollchor from 'react-scrollchor';
import Menu from '../src/components/menu';
import renderer from 'react-test-renderer';

describe('components', () => {
  describe('<Menu/>', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Menu />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
