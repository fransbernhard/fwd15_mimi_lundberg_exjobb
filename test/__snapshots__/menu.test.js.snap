// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <Menu/> renders correctly 1`] = `
<header>
  <nav>
    <ul>
      <li>
        <a
          href="#top"
          onClick={[Function]}
        >
          <a
            onClick={[Function]}
            style={Object {}}
          >
            Home
          </a>
        </a>
      </li>
      <li>
        <a
          href="#about"
          onClick={[Function]}
        >
          <a
            onClick={[Function]}
            style={Object {}}
          >
            About
          </a>
        </a>
      </li>
      <li>
        <a
          onClick={[Function]}
          style={Object {}}
        >
          Archive
        </a>
      </li>
      <li
        className="contactMobile"
      >
        <a
          href="mailto:magdamargaretha@gmail.com?Subject=magdenmagden"
        >
          Contact
        </a>
      </li>
      <li
        className="contactWeb"
      >
        <a
          href="#contact"
          onClick={[Function]}
        >
          <a
            onClick={[Function]}
            style={Object {}}
          >
            Contact
          </a>
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
